### PR TITLE
Remove swarm packages and related, fix unit tests

### PR DESCRIPTION
**Pull request by paologalligit**
This PR removes the entire swarm package since it's not used anymore. It also fixes the unit tests; this way a docker container can be created using the Dockerfile in the repo and used to run tests on github.

### CODE REVIEW

**Review by libotony — DISMISSED** [review]

File: .github/workflows/on-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Pull Request CI
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (libotony) [inline .github/workflows/on-pull-request.yaml]:
```suggestion
      - thor-upstream
```

**Comment by libotony** [comment]
@paologalligit @leszek-vechain appended 3 commits to make test pass

1. Removed fuzz test temporarily, since it's not compatible with go 1.10, needs to add it back in the future
2. Skipped mobile package test, it's not working in go 1.10, and we are not using it, skip for now, seek the possibility to enable it in the future
3. Removed incompatible test in parseNode, by incompatible I mean not working at go 1.10 but works in go 1.22

After the test and go module PR pass, our next goal might be seeking upgrading to most recent go version, for example go 1.22. But that might take more work to do since there is go API change from go 1.10 to go 1.22.

**Comment by leszek-vechain** [comment]
should we close this ticket then and open new one with upgrade to go 1.22 ?

**Comment by libotony** [comment]
> should we close this ticket then and open new one with upgrade to go 1.22 ?

I would suggest try it first, my previous try shows more places to be fixed.

**Review by libotony — APPROVED** [review]

**Comment by libotony** [comment]
Something wrong with my local GPG signing, let me see how to fix it

**Pull request merged**